crates/ruff_python_formatter/resources/test/fixtures/ruff/statement/function.py

```diff
@@ -231,3 +256,19 @@ def f42(
     c,
 ):
     pass
+
+
+# Type parameter comments
```

Comment:
Would you mind adding an example for an empty type param that contains comments

```suggestion
def type_param_empty_comments[ # trailing
	# own line 
]: pass
```

Reply:
Should we handle this? It's not valid syntax

```
❯ python3.12 -c "class Foo[]: ..."
  File "<string>", line 1
    class Foo[]: ...
              ^
SyntaxError: invalid syntax
```

Reply:
```
❯ python3.12 -c "def foo[](): ..."
  File "<string>", line 1
    def foo[](): ...
           ^
SyntaxError: expected '('
```